{"name": "test_auth[roma2073@rambler.ru-197323]", "status": "failed", "statusDetails": {"message": "AssertionError: assert not True\n +  where True = <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>>()\n +    where <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>> = <pages.elements.WebElement object at 0x000002337EBF3DC8>.is_visible\n +      where <pages.elements.WebElement object at 0x000002337EBF3DC8> = <pages.pages_auth.RTmis object at 0x000002337F768AC8>.error", "trace": "web_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ebb46d6d3d299f412b07de00981a9e4a\")>, login = 'roma2073@rambler.ru', password = '197323'\n\n    @allure.feature('Авторизация на https://rtmis.ru/')\n    @allure.story('Тест с авторизации с рандомизируемыми данными')\n    def test_auth(web_browser, login:str, password:str):\n        web = RTmis(web_browser)\n    \n        with allure.step(f\"Открываем {web.url}\"):\n          web.get(url=web.url)\n        with allure.step(\"Заполняем форму (логин:пароль)\"):\n          assert web.login.send_keys(login)\n          assert web.password.send_keys(password)\n        with allure.step(\"Нажимаем на кнопку войти\"):\n          web.auth.human_click()\n        with allure.step(\"Определям на странице сообщение об ошибке авторизации\"):\n>           assert not web.error.is_visible()\nE           assert not True\nE            +  where True = <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>>()\nE            +    where <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>> = <pages.elements.WebElement object at 0x000002337EBF3DC8>.is_visible\nE            +      where <pages.elements.WebElement object at 0x000002337EBF3DC8> = <pages.pages_auth.RTmis object at 0x000002337F768AC8>.error\n\ntests\\test_auth.py:18: AssertionError"}, "steps": [{"name": "Открываем https://demo.rtmis.ru/", "status": "passed", "start": 1643733530187, "stop": 1643733533662}, {"name": "Заполняем форму (логин:пароль)", "status": "passed", "start": 1643733533662, "stop": 1643733533782}, {"name": "Нажимаем на кнопку войти", "status": "passed", "start": 1643733533782, "stop": 1643733544155}, {"name": "Определям на странице сообщение об ошибке авторизации", "status": "failed", "statusDetails": {"message": "AssertionError: assert not True\n +  where True = <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>>()\n +    where <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>> = <pages.elements.WebElement object at 0x000002337EBF3DC8>.is_visible\n +      where <pages.elements.WebElement object at 0x000002337EBF3DC8> = <pages.pages_auth.RTmis object at 0x000002337F768AC8>.error\n", "trace": "  File \"C:\\Users\\ole lukoie\\Desktop\\ui\\tests\\test_auth.py\", line 18, in test_auth\n    assert not web.error.is_visible()\n"}, "start": 1643733544155, "stop": 1643733544210}], "parameters": [{"name": "login", "value": "'roma2073@rambler.ru'"}, {"name": "password", "value": "'197323'"}], "start": 1643733530187, "stop": 1643733544210, "uuid": "41c44c2f-6897-4d28-9d25-cbb28ae8ba6f", "historyId": "6f6d6a4308fce93f46a226cfd79c1cc4", "testCaseId": "5306df55395c93b83e69abe887b683bd", "fullName": "tests.test_auth#test_auth", "labels": [{"name": "feature", "value": "Авторизация на https://rtmis.ru/"}, {"name": "story", "value": "Тест с авторизации с рандомизируемыми данными"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "DESKTOP-2HIEP5G"}, {"name": "thread", "value": "1520-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}